WcaOnRails/spec/mailers/previews/competitions_mailer_preview.rb

```diff
@@ -7,6 +7,22 @@ def notify_board_of_confirmed_competition
     CompetitionsMailer.notify_board_of_confirmed_competition(c.delegates[0], c)
   end
 
+  def notify_organizer_of_confirmed_competition
+    c = CompetitionDelegate.last.competition
+    CompetitionsMailer.notify_organizer_of_confirmed_competition(c.delegates[0], c)
+  end
+
+  def notify_organizer_of_announced_competition
+    c = CompetitionDelegate.last.competition
+    p = "dummy_link"
```

Comment:
I am not sure if there is a nice way to link to the competition announcement, which is why I left this.